### DISCUSSION TITLE
Translate notification mail

### DISCUSSION THREAD

**Discussion by LordSpectre** [immich-app/immich#11276] [Q&A]
Hello, is there a way to translate notification mail from english to my own language? Both object and body. My old parents don't know english and they ignore them once received. Thanks

**Comment by ich777** [comment]
I would also appreciate that, that would be a great feature and a seamless experience.

Maybe the default language can be specified for email notifications, I mean in a perfect world it would be cool to use the users selected language but I think a global default is good enough.

**Reply by AtreideGET** [reply]
I would also appreciate that. My mother doesn't read english to.
